database/migrations/2021_05_13_042738_create_cases_table.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateCasesTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('cases', function (Blueprint $table) {
+            $table->id();
+            $table->integer('user_id');
+            $table->integer('province_id');
+            $table->integer('area_id');
+            $table->string('infection_type');
+            $table->integer('heal');
+            $table->integer('curing');
+            $table->integer('infection');
+            $table->integer('death');
+            $table->date('date');
```

Comment:
I think you should remove `date` because `$table->timestamps();` will generate two columns:
- `created_at`
- `updated_at`
So you can use `created_at` as input date.

Reply:
Yes

Reply:
yes